lib/unifex/code_generator/utils.ex

```diff
@@ -96,4 +96,9 @@ defmodule Unifex.CodeGenerator.Utils do
     |> Enum.map(&(&1 <> ";"))
     |> Enum.join("\n")
   end
+
+  @spec generate_maybe_unused_args_statements(args :: [String.t()]) :: [String.t()]
+  def generate_maybe_unused_args_statements(args) do
+    args |> Enum.map(fn arg -> ~g<UNIFEX_MAYBE_UNUSED(#{arg});> end)
+  end
```

Comment:
Now, this function seems an overkill, it's simpler just to write `UNIFEX_MAYBE_UNUSED(arg)` everywhere :P But you can leave it that way if it's ok for you

Reply:
Yeah, but it should be a little simpler to add more args as `maybe unused` in the future  :sweat_smile: